Makefile

```diff
@@ -5,9 +5,10 @@ PYTHON=/usr/bin/python3
 
 # check for executables
 
-PYFLAKES = $(shell { command -v pyflakes-3 || command -v pyflakes3 || command -v pyflakes; }  2> /dev/null)
-
-PYCODESTYLE := $(shell { command -v pycodestyle-3 || command -v pycodestyle3 || command -v pycodestyle; } 2> /dev/null)
+PYFLAKES = $(shell which pyflakes)
```

Comment:
You might better revert this,since it is breaking the build for Debian. Debian "wants" `pyflakes3`.

Reply:
Fixed through 7d90c8d